assembl/views/api2/discussion.py

```diff
@@ -279,54 +281,55 @@ def get_time_series_analytics(request):
-                     ).label('retention_count_last_visit_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.first_visit == None, 0),
-                      (and_(subscribersAgentStatus.first_visit < intervals_table.c.interval_end,
-                        subscribersAgentStatus.first_visit >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('recruitment_count_first_visit_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.first_subscribed == None, 0),
-                      (and_(subscribersAgentStatus.first_subscribed < intervals_table.c.interval_end,
-                        subscribersAgentStatus.first_subscribed >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('recruitment_count_first_subscribed_in_period'),
-            func.sum(
-                case([
-                      (subscribersAgentStatus.last_unsubscribed == None, 0),
-                      (and_(subscribersAgentStatus.last_unsubscribed < intervals_table.c.interval_end,
-                        subscribersAgentStatus.last_unsubscribed >= intervals_table.c.interval_start), 1)
-                      ], else_=0)
-                     ).label('retention_count_last_unsubscribed_in_period'),
-        )
-        subscribers_query = subscribers_query.outerjoin(subscribersAgentStatus, subscribersAgentStatus.discussion_id==discussion.id)
+                                                func.sum(
+                                                    case([
+                                                        (subscribersAgentStatus.last_visit == None, 0),
+                                                        (and_(subscribersAgentStatus.last_visit < intervals_table.c.interval_end,
+                                                              subscribersAgentStatus.last_visit >= intervals_table.c.interval_start), 1)
+                                                    ], else_=0)
+                                                ).label('retention_count_last_visit_in_period'),
+                                                func.sum(
+                                                    case([
+                                                        (subscribersAgentStatus.first_visit == None, 0),
+                                                        (and_(subscribersAgentStatus.first_visit < intervals_table.c.interval_end,
+                                                              subscribersAgentStatus.first_visit >= intervals_table.c.interval_start), 1)
+                                                    ], else_=0)
+                                                ).label('recruitment_count_first_visit_in_period'),
+                                                func.sum(
+                                                    case([
```

Comment:
Next time, please have separate commit that contains only changes in spacing (not combined with regular code edits)

Reply:
sorry about that